package.json

```diff
@@ -237,10 +237,12 @@
     "electron-log": "^2.2.17",
     "electron-updater": "^4.0.6",
     "history": "^4.7.2",
+    "prop-types": "^15.7.2",
```

Comment:
Can you remove `package-lock.json` and fix the out-of-sync `yarn.lock`? Project is supposed to be used with `yarn`, thus there's no need for the second lock file:

```
 \ $ yarn-clean-install
yarn install v1.15.2
warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.
```